lib/picoruby/build.rb

```diff
@@ -40,6 +40,10 @@ def picoruby(alloc_libc: true)
         cc.defines << define if cc.defines.none? { _1.start_with? key }
       end
 
+      cc.defines << "PICORUBY_COMMIT_TIMESTAMP=#{`git log -1 --format=%cd --date=iso-strict`.strip}"
```

Comment:
```suggestion
      cc.defines << "PICORUBY_COMMIT_TIMESTAMP=#{`git log -1 --format="%ct" | awk '{print strftime("%Y-%m-%dT%H:%M:%SZ", $1, 1)}'`.strip}"
```

That way you use produces local time. It should be UTC

Reply:
Also, I suspect that even if a new commit is made, `version.o` will not be updated because `version.c` has not been changed, and as a result the version information will not be updated.

Reply:
The awk command installed by default in macOS cannot use strftime. How about using the xargs and date commands as follows?

```sh
git log -1 --format=%ct | xargs -I{} date -u -r {} '+%Y-%m-%dT%H:%M:%SZ'
```

Reply:
In the release process, `rake clean` -> `rake` will ensure that the correct version is written.
Is it not a big problem if the correct version is not always written during development?

Reply:
> git log -1 --format=%ct | xargs -I{} date -u -r {} '+%Y-%m-%dT%H:%M:%SZ'

That doesn't work on Ubuntu because `-r` option is incompatible between BSD date and GNU date.
We can use Ruby instead:
```
Time.at(`git log -1 --format=%ct`).strftime('%Y-%m-%dT%H:%M:%SZ')
```

> In the release process, rake clean -> rake will ensure that the correct version is written.
> Is it not a big problem if the correct version is not always written during development?

I don't think so. I will refer to the version info during development. I want it to be always correct